### PR TITLE
feat: include new relayer address override

### PR DESCRIPTION
**Pull request by james-a-morris**
We need to be mindful of Synthetix's relayer when dealing with OP/Homestead routes of SNX. Specifically, our quoting should by default come from SNX's relayer unless specified otherwise where the SNX token <-> 10 or 1 is concerned.

### CODE REVIEW

**Review by dohaki — APPROVED** [review]
Nice LGTM

File: api/_constants.ts [hunk 1]
```diff
@@ -95,6 +95,16 @@ const defaultRelayerFeeCapitalCostConfig: {
   },
 };
 
+export const defaultRelayerAddressOverride: Record<
+  string,
+  { relayer: string; destinationChains: number[] }
+> = {
+  SNX: {
+    relayer: "0x19cdc2b23af0cc791ca64dda5bfc094cddda31cd",
```
Comment (nicholaspai) [inline api/_constants.ts:103]: This should be checksummed right?

Comment (james-a-morris) [reply api/_constants.ts]: We don't directly compare against these addresses - but I agree it's more sanitized to checksum the address.

**Review by nicholaspai — APPROVED** [review]
LGTM

**Pull request merged**